### PR TITLE
Updating Contact page with job postings and job opportunities in TISLab

### PR DESCRIPTION
**Pull request by tursynay**
I have checked that:

- [ ] all pages' content, images, and styles still appear correctly
- [ ] all links still work
- [ ] redirects (e.g `deploy-preview-XX--tislab-website.netlify.app/melzoom`) are still working

### CODE REVIEW

**Comment by tursynay** [comment]
@jmcmurry , would you take a look and see if this look ok? I want to remove the Google Map thing on this page as it is not a real TISLab location but looks like placeholder, tried to comment out, did not work, in will be make another PR?

File: contact/index.md [hunk 1]
```diff
@@ -20,15 +18,15 @@ Our lab is based at the [University of North Carolina's Genetics Department](htt
...
   style="button"
 %}
-{%
+<!{%
   include link.html
   type="address"
   icon=""
   text="Google Maps"
   tooltip="Our location on Google Maps for easy navigation"
   link="https://www.google.com/maps/place/That+St+%26+The+Other+St,+Porters+Lake,+NS+B3E+1H3,+Canada/@44.7389237,-63.3033296,20.78z/data=!4m5!3m4!1s0x4b5a31023bb02565:0xb9505694e83a53d7!8m2!3d44.7389353!4d-63.3030828"
```
Comment (jmcmurry) [inline contact/index.md]: This should be https://www.google.com/maps/place/Genetic+Medicine+Building/@35.9014048,-79.0543985,17z/data=!3m1!4b1!4m6!3m5!1s0x89acc3903cfccd29:0xa3012e6cb634b2d5!8m2!3d35.9014048!4d-79.0543985!16s%2Fg%2F11trh7sh82?entry=ttu

File: contact/index.md [hunk 2]
```diff
@@ -37,3 +35,30 @@ Our lab is based at the [University of North Carolina's Genetics Department](htt
   width="400px"
   link="https://www.med.unc.edu/genetics/"
 %}
+
+**Join Our Lab Team! Exciting Job Opportunities Await!**
```
Comment (jmcmurry) [inline contact/index.md:39]: This part looks great!

File: contact/index.md
Comment (jmcmurry) [inline contact/index.md:1]: Please update the maps link thanks :)

**Review by jmcmurry — CHANGES_REQUESTED** [review]

File: contact/index.md
Comment (tursynay) [reply contact/index.md:1]: thank you

File: contact/index.md (hunk 1 above)
Comment (tursynay) [reply contact/index.md]: updated

**Comment by tursynay** [comment]
@jmcmurry , updated, I can merge or you can merge

**Pull request merged**